core/conversion/converters/impl/element_wise.cpp

```diff
@@ -180,6 +181,51 @@ auto element_wise_registrations TRTORCH_UNUSED = RegisterNodeConversionPatterns(
                 mul->setName(util::node_info(n).c_str());
                 auto out = ctx->AssociateValueAndTensor(n->outputs()[0], mul->getOutput(0));
 
+                LOG_DEBUG("Output tensor shape: " << out->getDimensions());
+                return true;
+             }
+         }).pattern({
+            "aten::pow.Tensor_Tensor(Tensor self, Tensor exponent) -> (Tensor)",
+            [](ConversionCtx* ctx, const torch::jit::Node* n, args& args) -> bool {
+                // TODO: Remove with functionalization
+                auto self = args[0].ITensorOrFreeze(ctx);
+                auto exponent = args[1].ITensorOrFreeze(ctx);
+                auto pow = add_elementwise(ctx, nvinfer1::ElementWiseOperation::kPOW, self, exponent, util::node_info(n));
+                TRTORCH_CHECK(pow, "Unable to create Power layer from node: " << *n);
+
+                pow->setName(util::node_info(n).c_str());
+                auto out = ctx->AssociateValueAndTensor(n->outputs()[0], pow->getOutput(0));
+
+                LOG_DEBUG("Output tensor shape: " << out->getDimensions());
+                return true;
+             }
+         }).pattern({
+            "aten::pow.Tensor_Scalar(Tensor self, Scalar exponent) -> (Tensor)",
+            [](ConversionCtx* ctx, const torch::jit::Node* n, args& args) -> bool {
+                auto self = args[0].ITensorOrFreeze(ctx);
+                auto exponentScalar = args[1].unwrapToScalar().to<float>();
+                
+                // Calculate size of the input and define an exponent tensor of the same size
+                int volume=1;
+                for(int i=0; i<self->getDimensions().nbDims; i++)
+                    volume = volume*(self->getDimensions().d[i]);
+                
+                // Create a torch tensor with constant exponent values
+                LOG_DEBUG("Broadcasting the exponent in power layer");
+                std::vector<float> exponentVec(volume, exponentScalar);
+                torch::Tensor exponentBlob = torch::from_blob(exponentVec.data(), exponentVec.size());
```

Comment:
I think there is a torch API to do this directly. Issue here is torch wont own the memory so it may have some adverse effects. I think its `torch::fill`

Reply:
Had some issues before. Now changed it to use torch::full. Thanks